components/patient-access-rules/ui/src/main/resources/PhenoTips/AccessRightsManagement.xml

```diff
@@ -1479,7 +1479,7 @@ $xwiki.ssx.use($serviceDocName)
     #if ($firstCollaborator.isUser())
       #set ($collaboratorName = $xwiki.getUserName("$firstCollaboratorReference", false))
     #else
-      #set ($collaboratorName = $xwiki.getDocument($firstCollaboratorReference).plainTitle)
+      #set ($collaboratorName = $xwiki.getDocument($firstCollaboratorReference).getDisplayTitle())
```

Comment:
Why this change?

Reply:
`plainTitle` is un-escaped for group name, unlike `getDisplayTitle()`. I had it originally as `$escapetool.html($xwiki.getDocument($firstCollaboratorReference).plainTitle)` right before insertion which also works.

With you're previous comment, following ALAP, perhaps the better place to escape would be right before HTML insertion (Line 1484) and keep `plainTitle`. I ended up putting it here since `$collaboratorName` is only used for insertion and it felt like I was hiding an escape at insertion.

Reply:
However, it might be wise to start using `getDisplayTitle()` which returns a string already escaped. Maybe @marta-  can give a suggestion?

Reply:
After some discussion, I'll change it back to `plainTitle` and follow ALAP convention. Thanks for the help.